include/database.py

```diff
@@ -59,3 +59,69 @@ def insert_returning(connectionString, sql, parameters):
     except psycopg2.DatabaseError as err:
         sys.stderr.write(f'A general database error occurred: {err}')
         raise DatabaseError
+
+
+def remove_record(connectionString, sql, parameters):
+    try:
+        # Open the connection, override any timeout provided with something shorter
+        # since we expect to be running interactively
+        connection = psycopg2.connect(connectionString, connect_timeout=1)
+        cursor = connection.cursor()
+
+        # Execute the query, note the rows
+        cursor.execute(sql, (parameters['id'],))
```

Comment:
You should be able to just pass the contents of `parameters` since they should be formatted correctly for the SQL